src/mantine-hooks/src/use-hash/use-hash.ts

```diff
@@ -2,7 +2,7 @@ import { useState } from 'react';
 import { useWindowEvent } from '../use-window-event/use-window-event';
 
 export function useHash(): readonly [string | null, (hash: string) => void] {
-  const [hash, setHashValue] = useState<string>(null);
+  const [hash, setHashValue] = useState<string>(window.location.hash);
```

Comment:
It will not work, window object is not accessible during ssr, thus hook will throw an error

Reply:
Oh wasn't thinking about backend rendering... maybe we can run the set function under an effect?

Reply:
Hook must check the environment, see any other hook that uses window, e.g. https://github.com/mantinedev/mantine/blob/master/src/mantine-hooks/src/use-viewport-size/use-viewport-size.ts

Reply:
Sorry for disappearing, I accidentally spilled an entire cup of coffee on my mac :(
I added the check for window object, and also added optional chaining if needed.

Would this work?

Reply:
Compile failed due to prettier, I am currently laptop-less, so can only do changes in the browser...
Feel free to close this one while my laptop is getting repaired